pkg/kubectl/describe.go

```diff
@@ -775,18 +790,45 @@ func describeContainers(pod *api.Pod, out io.Writer) {
 	}
 }
 
-func envValueFrom(pod *api.Pod, e api.EnvVar) string {
-	internalFieldPath, _, err := api.Scheme.ConvertFieldLabel(e.ValueFrom.FieldRef.APIVersion, "Pod", e.ValueFrom.FieldRef.FieldPath, "")
-	if err != nil {
-		return "" // pod validation should catch this on create
+// DescribeProbe is exported for consumers in other API groups that have probes
+func DescribeProbe(probe *api.Probe) string {
+	attrs := fmt.Sprintf("delay=%ds timeout=%ds period=%ds #success=%d #failure=%d", probe.InitialDelaySeconds, probe.TimeoutSeconds, probe.PeriodSeconds, probe.SuccessThreshold, probe.FailureThreshold)
+	switch {
+	case probe.Exec != nil:
+		return fmt.Sprintf("exec %v %s", probe.Exec.Command, attrs)
```

Comment:
This will print Command with brackets since it's a slice.
https://play.golang.org/p/mKPuASjQPl
Do we want that?

Also, how about a separate line for the attributes?